example/README.md

```diff
@@ -0,0 +1,37 @@
+# Script loader example
```

Comment:
I'm sry, but we try to avoid adding (new) `example/` to repos directly as they tend to be getting outdated and nobody is normally willing to maintain them. Please add a small example to the README instead (as mentiond above) and remove the `example/` folder